UPGRADE.md

```diff
@@ -100,6 +100,125 @@
    +                       class: "%sylius.model.channel.class%"
    #           ...
    ```
+   
+1. Service and Alias Refactoring for [New Naming Convention](https://github.com/Sylius/Sylius/blob/2.0/adr/2024_10_03_services_naming_convention.md)
```

Comment:
This title and ADR applies only to services changes, but there are also routes, parameters, etc. It would be nice to split it